lib/vfscore/Config.uk

```diff
@@ -2,7 +2,7 @@ config LIBVFSCORE
 	bool "vfscore: VFS Core Interface"
 	default n
 	select LIBNOLIBC if !HAVE_LIBC
-	select LIBUKTIME if !HAVE_LIBC
```

Comment:
Shouldn't `libposix-time` always be selected if vfscore really needs it (remove !HAVE_LIBC)? and do really all libc's provide its functionality?

Reply:
I removed them from anywhere